src/Persistence_SQL.php

```diff
@@ -127,6 +127,23 @@ public function initQueryFields($m, $q)
         }
     }
 
+    protected function setLimitOrder($m, $q)
+    {
+        if ($m->limit && ($m->limit[0] || $m->limit[1])) {
+            if ($m->limit[0] === null) {
+                // really, SQL?
+                $m->limit[0] = '18446744073709551615';
```

Comment:
it's a stupid SQL limitation, and that number shouldn't be there in the frist place. 

See http://stackoverflow.com/questions/255517/mysql-offset-infinite-rows